domain/src/main/kotlin/feature/ExportManager.kt

```diff
@@ -13,24 +13,36 @@ import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.runBlocking
 import ltd.evilcorp.core.repository.MessageRepository
 import ltd.evilcorp.core.vo.Message
+import org.json.JSONArray
+import org.json.JSONObject
 
 class ExportManager @Inject constructor(
     private val messageRepository: MessageRepository,
 ) {
-    // TODO @Akito: Increment version programmatically on major changes.
-    private fun List<Message>.generateExportMessages(): ExportMessages {
-        return ExportMessages(
-            version = 1,
-            timestamp = SimpleDateFormat(
-                """yyyy-MM-dd'T'HH-mm-ss""",
-                Locale.getDefault(),
-            ).format(Date()),
-            entries = this,
+    private fun generateExportMessagesJString(messages: List<Message>): String {
+        val root = JSONObject()
+        root.put("version", 1)
```

Comment:
At some point in the future, when needed, we need to make this version more flexible, rather than changing this hard code, here. Just FYI for the future.